cogs/mod.py

```diff
@@ -21,32 +21,38 @@
 from discord.ext import commands
 import asyncio
 
+from .utils.checks import is_moderator
+
 
 class Mod():
     def __init__(self, bot):
         self.bot = bot
 
-    async def on_message(self, message):
-        if message.author == self.bot.user:
-            return
-        ctx = await self.bot.get_context(message)
-        if ctx.command:
-            return
-        if isinstance(message.channel, discord.DMChannel):
-            channel_to_send = self.bot.get_channel(454061583874785280)
-            msg = '{} 📣 {}'.format(str(message.author), message.content)
-            await channel_to_send.send(msg)
+    @commands.command()
+    async def answer(self, ctx, *args):
```

Comment:
I need a mod's say on this change @mjip

Reply:
(This was discussed in #idoneam with @jidicula)